### DISCUSSION TITLE
[Bug] Wrong collection time

### DISCUSSION THREAD

**Discussion by yinbucheng** [apache/skywalking#9353] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Java Agent (apache/skywalking-java)

### What happened

![image](https://user-images.githubusercontent.com/30992273/179658588-451963ec-a9c3-4d91-bef2-7f548f0085a7.png)

The previous step takes less time than the sub step

oap和ui：9.1.0（最新）
agent：8.11.0（最新）

### What you expected to happen

The previous step takes more time than the sub step or equal to

### How to reproduce

oap和ui：9.1.0（最新）
agent：8.11.0（最新）

feign invoke 

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
I am not sure whether this is a bug, you should check the reported spans/segments. Generally time gaps could be caused by async processes.

**Reply by yinbucheng** [reply]
The next call to Kafka adopts asynchronous mode, and the others are synchronous，Does the time consumption include the time consumed by asynchronous execution？

The following is our own monitoring platform. It shows that this interface takes time, excluding the transmission time of the response to the upstream service network
![image](https://user-images.githubusercontent.com/30992273/179661920-5ebf799a-11f7-42e9-8cf3-80e8a78f8280.png)

Here is the monitoring time of skywalking
![image](https://user-images.githubusercontent.com/30992273/179661994-b63cff1f-8f73-4741-97a8-34108a7df82e.png)

Besides, I didn't find anything like spanid and traceid in the details
![image](https://user-images.githubusercontent.com/30992273/179662421-732080c7-640a-4f54-bce1-18dc9a8c911b.png)

**Reply by yinbucheng** [reply]
How can I cooperate with you to locate this. My operation is very simple. Query mongodb and send Kafka messages

**Reply by wu-sheng** [reply]
I don't think I have anything to do. 
The data is at your side, the plugin is running in your codes. 
You could debug codes to find span's fields and data.

**Reply by yinbucheng** [reply]
![image](https://user-images.githubusercontent.com/30992273/179701231-d088c5b6-164b-44de-931f-65015b627677.png)
![image](https://user-images.githubusercontent.com/30992273/179701575-ac8918f9-4a5f-443d-8c85-036fec673ea3.png)

This problem is not inevitable，90% of my requests are corresponding。

**Reply by yinbucheng** [reply]
Is it related to APM collecting data and reporting OAP？

**Reply by wu-sheng** [reply]
To be fair, I can't know. You need to find out. 
This mostly is relative to an agent plugin. I am not sure.
